app/views/pages/home.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="initial-scale=1, width=device-width" />
+ </head>
+  <body>
+   <div class="homepage">
+      <img class="homepage-image" alt="" src="https://images.unsplash.com/photo-1466428996289-fb355538da1b?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1170&q=80" />
+      <div class="heading">
+           <div class="homepage-title">Welcome to MoodMaestro!</div>
+       </div>
+       <button class="button button1">Create a new playlist</button>
```

Comment:
This button has no function, use the link_to helper to point it to the actual route for it.